.github/workflows/ci.yml

```diff
@@ -101,69 +101,40 @@ jobs:
-          key: "php${{ matrix.php-version }}-typo3${{ matrix.typo3-version }}-${{ matrix.composer-dependencies }}-composer-${{ hashFiles('**/composer.json') }}"
-          path: ~/.cache/composer
-          restore-keys: "php${{ matrix.php-version }}-typo3${{ matrix.typo3-version }}-${{ matrix.composer-dependencies }}-composer-\n"
-      - name: Install TYPO3 Core
-        env:
-          TYPO3: "${{ matrix.typo3-version }}"
-        run: |
-          composer require --no-ansi --no-interaction --no-progress --no-install typo3/cms-core:"$TYPO3"
-          composer show
-      - name: Install lowest dependencies with composer
-        if: "matrix.composer-dependencies == 'lowest'"
+      - name: Install composer dependencies
         run: |
-          composer update --no-ansi --no-interaction --no-progress --with-dependencies --prefer-lowest
-          composer show
-      - name: Install highest dependencies with composer
-        if: "matrix.composer-dependencies == 'highest'"
-        run: |
-          composer update --no-ansi --no-interaction --no-progress --with-dependencies
-          composer show
+          Build/Scripts/runTests.sh -p ${{ matrix.php-version }} -t ${{matrix.typo3-version}} -s composerUpdate${{matrix.composer-dependencies}}
       - name: Run unit tests
-        run: "composer ci:tests:unit"
+        run: |
+          Build/Scripts/runTests.sh -p ${{ matrix.php-version }} -s unit
     strategy:
       fail-fast: false
       matrix:
         include:
-          - typo3-version: "^12.4"
+          - typo3-version: "12.4"
             php-version: "8.1"
-            composer-dependencies: lowest
-          - typo3-version: "^12.4"
+            composer-dependencies: Min
+          - typo3-version: "12.4"
             php-version: "8.1"
-            composer-dependencies: highest
-          - typo3-version: "^12.4"
+            composer-dependencies: Max
+          - typo3-version: "12.4"
             php-version: "8.2"
-            composer-dependencies: lowest
-          - typo3-version: "^12.4"
+            composer-dependencies: Min
+          - typo3-version: "12.4"
             php-version: "8.2"
-            composer-dependencies: highest
-          - typo3-version: "^12.4"
+            composer-dependencies: Max
+          - typo3-version: "12.4"
             php-version: "8.3"
-            composer-dependencies: lowest
-          - typo3-version: "^12.4"
+            composer-dependencies: Min
+          - typo3-version: "12.4"
             php-version: "8.3"
-            composer-dependencies: highest
-          - typo3-version: "^12.4"
+            composer-dependencies: Max
+          - typo3-version: "12.4"
             php-version: "8.4"
-            composer-dependencies: lowest
-          - typo3-version: "^12.4"
+            composer-dependencies: Min
+          - typo3-version: "12.4"
             php-version: "8.4"
-            composer-dependencies: highest
+            composer-dependencies: Max
```

Comment:
I propose we do this change (Min/Max and TYPO3 versions without `^` in the keys) across the board in a pre-patch to keep this PR here smaller.